src/components/utils/MapStateContext.tsx

```diff
@@ -39,6 +41,7 @@ export const MapStateProvider = ({ children, initialMapView }) => {
   const [bbox, setBbox] = useState();
   const [view, setView] = useState(initialMapView);
   const [viewForMap, setViewForMap] = useState(initialMapView);
+  const [showClimbing, setShowClimbing] = useState(false);
```

Comment:
Could we store this in the `layers` array instead? It is already persisted and eventually can come to URL.